nextflow.config

```diff
@@ -120,6 +127,7 @@ profiles {
     singularity.enabled = true
   }
   test { includeConfig 'conf/test.config' }
+  test_local { includeConfig 'conf/test_local.config' }
```

Comment:
```suggestion
```
I would remove the `conf/test_local.config` file.
I can see that it was useful for you, but I would prefer to add the option to do some local test with offline data in a different PR

Reply:
I erroneously left in there.  I couldn't think of a neat way of ignoring those changes

Reply:
Can you remove this line and the corresponding file?